string plan_name
---
moveit_msgs/RobotTrajectory moveit_plan
bool success
